client/src/components/JurisdictionAdmin/DownloadBatchTallySheetsButton/DownloadBatchTallySheetsButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import { Button } from '@blueprintjs/core'
+import { Document, pdf } from '@react-pdf/renderer'
+import { toast } from 'react-toastify'
+import * as FileSaver from 'file-saver'
+import * as Sentry from '@sentry/react'
+
+import BatchTallySheet from './BatchTallySheet'
+import useContestsJurisdictionAdmin from '../useContestsJurisdictionAdmin'
+import { useBatches } from '../useBatchResults'
+
+const FILE_NAME = 'batch-tally-sheets.pdf'
```

Comment:
This doesn't really match the style of some of the other file names, but they aren't super consistent so I think it's fine. Added a follow up task to fix: https://github.com/votingworks/arlo/issues/1479

Reply:
Nice, thanks for creating the follow-up task!